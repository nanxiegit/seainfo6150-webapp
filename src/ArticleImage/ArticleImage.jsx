import React from "react";
import PropTypes from "prop-types";
import styles from "./ArticleImage.module.css";

const ArticleImage = (props) => {
    return (
      <img className={styles.imgStyle} alt = {props.title} src = {props.url}/>

    );
  };
  
  ArticleImage.propTypes = {
    title: PropTypes.string.isRequired,
    url: PropTypes.string.isRequired,
  };
  export default ArticleImage;