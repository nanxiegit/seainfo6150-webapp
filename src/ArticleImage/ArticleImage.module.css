.imgStyle{
    background:red;
    border: 1px solid red;
    border-radius: 50%;
    box-shadow: 0px 0px 10px 5px rgba(204, 0, 0, 0.5);
    padding: 4px;
    height: 60%;
    width:20%
}