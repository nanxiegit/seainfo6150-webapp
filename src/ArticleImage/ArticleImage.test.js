import React from "react";
import ArticleImage from "./ArticleImage";

describe("ArticleImage tests", () => {
  it("renders correctly", () => {
    const { container } = render(<ArticleImage title={"You can take a pie on a plane"} 
      url={"https://cdn.cnn.com/cnnnext/dam/assets/170321135754-airport-security-line-file-super-169.jpg"} />);
    expect(container).toMatchSnapshot();
  });
});