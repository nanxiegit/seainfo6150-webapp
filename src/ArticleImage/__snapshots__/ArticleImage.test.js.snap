// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <img
    alt="You can take a pie on a plane"
    class="image"
    src="https://cdn.cnn.com/cnnnext/dam/assets/170321135754-airport-security-line-file-super-169.jpg"
  />
</div>
`;
